.gitignore

```diff
@@ -0,0 +1,211 @@
+# Created by https://www.toptal.com/developers/gitignore/api/maven,visualstudiocode,intellij+all,sublimetext,gradle,vim
```

Comment:
I'm no big fan of those huge .gitignore files. Most of this stuff is in my global_ignore as it is environment specific.
Why not just ignore .idea/ and remove all other stuff? Only keep lines 87, 89 and 90 for intelliJ stuff and remove the rest of the .idea/** entries

Reply:
I reduced it to the absolute minimum (VSCode and IntelliJ) now